dependencies.yaml

```diff
@@ -41,6 +41,12 @@ files:
     includes:
       - develop
       - py_version
+  docs:
+    output: none
+    includes:
```

Comment:
Have we verified that all the notebooks rendered during our docs build can execute with only the runtime dependencies of cudf? (Reminder to self, double check this.)

Reply:
The notebooks render correctly.